src/main/resources/citizen/templates/document/claimantResponseReceipt.html

```diff
@@ -314,13 +312,14 @@
           {{ courtDecisionrepaymentPlan.paymentSchedule }}
         </td>
       </tr>
+      {% endif %}
+
       <tr class="no-border">
-        <td><strong>Make the first payment on:</strong></td>
+        <td><strong>Last payment by:</strong></td>
         <td>
-          {{ courtDecisionrepaymentPlan.firstPaymentDate }}
+          {{ courtDecisionwhenWillTheyPay }}
```

Comment:
courtDecisionwhenWillTheyPay does not make sense here. Should it be courtDecisionwhenWillTheyFinishPaying?

Reply:
Fixed!